.github/workflows/test-application.yaml

```
@@ -51,6 +62,16 @@ jobs:
                     xpack.security.enabled: 'false'
                 options: --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=5
 
+            elasticsearch7:
+                image: docker.elastic.co/elasticsearch/elasticsearch:7.11.1
+                ports:
+                    - 9201:9200
+                env:
+                    discovery.type: 'single-node'
+                    xpack.security.enabled: 'false'
+                    ES_JAVA_OPTS: '-Xms512m -Xmx512m'
+                options: --health-cmd="curl http://localhost:9201/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=5
```

the health check goes on the internal not on the external port so here should be `9200`

```suggestion
                options: --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=5
```